theofidry/psysh-bundle/3.3/config/packages/psysh.yaml

```diff
@@ -0,0 +1,6 @@
+psysh:
+    variables:
+        container: '@service_container'
+        kernel: '@kernel'
+        self: '@psysh.shell'
+        parameters: "@=service('service_container').getParameterBag().all()"
```

Comment:
If I understand correctly, those variables are always defined, right? If that's the case, they should be removed here.

Reply:
So the practice is to have those config files for non-default values?